src/widget/wtracktableview.cpp

```diff
@@ -1424,9 +1424,8 @@ void WTrackTableView::slotExportTrackMetadataIntoFileTags() {
         QMessageBox::information(
             nullptr,
             tr("Export Track Metadata"),
-            tr("File modifications are deferred and may not appear at once! "
-                    "If you do not see changed metadata in other programs, "
-                    "close Mixxx to modify those files immediately."));
+            tr("Mixxx may wait to modify files until it is certain that writing into those files will not cause audible glitches! "
```

Comment:
This should reference the same string constant as DlgPrefLibrary so translators do not translate identical strings twice.

Reply:
@esbrandt can you comment on this? Does that present an issue for translators to have the exact same string repeated in different places in the code?

Reply:
100 % identical strings gets auto-filled by the translation memory, if one occurrence of string is manually translated.
For anything lower than 100% the translator is presented a list of suggestions, sorted by similarity, and with the possibility to compare differences.

tl;dr No issue, and preferable